locale/en/blog/weekly-updates/weekly-update.2015-11-20.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Weekly Update - Nov 20th, 2015
+author: Minwoo Jung (@jmwsoft)
+date: 2015-11-20T12:00:00.000Z
+status: publish
+category: weekly
+slug: weekly-update-2015-11-20
+layout: blog-post.hbs
+---
+
+### Node.js News — November 20th
+Node.js v5.1.0 (Stable) is released
+
+### Node.js v5.1.0 (Stable) Releases
+
+This week we have one release: [Node.js v5.1.0 (Stable)](https://nodejs.org/en/blog/release/v5.1.0/). Complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md).
+
+### Notable changes
+
+* **doc**: All of the API docs have been re-ordered so as to read in alphabetical order (Tristian Flanagan) [#3662](https://github.com/nodejs/node/pull/3662).
+* **http_parser**: update http-parser to 2.6.0 from 2.5.0 (James M Snell) [#3569](https://github.com/nodejs/node/pull/3569).
+  - Now supports the following HTTP methods: `LINK`, `UNLINK`, `BIND`, `REBIND`, `UNBIND`.
+  - Also added ACL and IPv6 Zone ID support.
+* **npm**: upgrade npm to 3.3.12 from v3.3.6 (Rebecca Turner) [#3685](https://github.com/nodejs/node/pull/3685).
+  - See the release notes for [v3.3.7](https://github.com/npm/npm/releases/tag/v3.3.7)
```

Comment:
If it's update to 3.3.12, shouldn't those be the release notes pointed to and not 3.3.7? Or was it actually updated to 3.3.7?